pkg/repository/manager.go

```diff
@@ -90,30 +89,14 @@ type Manager struct {
 }
 
 // LoadRepository loads information about a single local repository
-func (m *Manager) LoadRepository(name string, t RepositoryType) (*Repository, error) {
+func (m *Manager) LoadRepository(name string) (*Repository, error) {
 	// check repo cache
-	repo, err := m.GetRepository(name)
+	repo, err := m.getRepository(name)
 	if err == nil {
 		return repo, nil
 	}
 
-	// fail if repository doesn't exist in the manager configuration
-	switch t {
-	case RepositoryTypeCore:
-		if !util.InStrings(name, m.cfg.Repositories.Core) {
-			return nil, ErrUnconfiguredRepository
-		}
-	case RepositoryTypeController:
-		if !util.InStrings(name, m.cfg.Repositories.Services) {
-			return nil, ErrUnconfiguredRepository
-		}
-	}
```

Comment:
unfortunatelly this have broken some other commands/tests - sending a follow up PR